winml/lib/Api.Ort/OnnxruntimeModel.cpp

```diff
@@ -47,7 +47,7 @@ HRESULT CreateFeatureDescriptors(
   return S_OK;
 }
 
-HRESULT ModelInfo::RuntimeClassInitialize(OnnxruntimeEngineFactory* engine_factory, OrtModel* ort_model) {
+HRESULT ModelInfo::RuntimeClassInitialize(OnnxruntimeEngineFactory* engine_factory, _In_ OrtModel* ort_model) {
```

Comment:
why doesn't "OnnxruntimeEngineFactory" need an annotation?

Reply:
Following the existing interface annotation schema. My expectation is because engine_factory is not immutable.

Reply:
You can add one.  And I think  `_In_` is for const pointers , like  const OrtModel*.  `_Inout_` is used to annotate a pointer parameter that may be changed by the function.

e.g. 
```c++
void InOutCallee(_Inout_ int *pInt)
{
   int i = *pInt;
   *pInt = 6;
}

```
see: https://docs.microsoft.com/en-us/cpp/code-quality/understanding-sal?view=msvc-160

Reply:
That's true I could use the _Inout_ param, my guess is that the EngineFactory can be modified via its sharedpointer to an OrtEnvironment.

Reply:
Oh I misread the code, and didn't realize that the declaration of ModelInfo::RuntimeClassInitialize had an _in_ annotation on engine_factory.